### PR TITLE
Adding --features nightly tests to NayDuck

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

**Review by bowenwang1996 — COMMENTED** [review]
will results be collected separately in nayduck?

**Comment by utka** [comment]
@bowenwang1996 It will be under one run, but with separate build types. If you go to NayDuck, search for Debug/Nightly BUILD DONE, you will see the example.

**Review by bowenwang1996 — APPROVED** [review]

File: nightly/complete_nayduck_set.txt [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+pytest adversarial/start_from_genesis.py --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py overtake
+pytest adversarial/start_from_genesis.py overtake --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py doomslug_off
+pytest adversarial/start_from_genesis.py doomslug_off --features nightly_protocol --features nightly_protocol_features
+pytest adversarial/start_from_genesis.py overtake doomslug_off
+pytest adversarial/start_from_genesis.py overtake doomslug_off --features nightly_protocol --features nightly_protocol_features
+
+# python test for enable inflation
+pytest sanity/enable_inflation.py
```
Comment (bowenwang1996) [inline nightly/complete_nayduck_set.txt:151]: this test is no longer needed. Let's remove it

File: nightly/complete_nayduck_set.txt [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+pytest sanity/enable_inflation.py --features nightly_protocol --features nightly_protocol_features
+
+# python upgradable test
+# upgradable.py moves `near` binary, and must be the last python test in the set
+pytest --timeout=600 sanity/upgradable.py
+pytest --timeout=600 sanity/upgradable.py --features nightly_protocol --features nightly_protocol_features
```
Comment (bowenwang1996) [inline nightly/complete_nayduck_set.txt:157]: nightly isn't always backward compatible so we should not enable nightly features for this test

File: nightly/complete_nayduck_set.txt (hunk 1 above)
Comment (utka) [reply nightly/complete_nayduck_set.txt]: Done

File: nightly/complete_nayduck_set.txt (hunk 2 above)
Comment (utka) [reply nightly/complete_nayduck_set.txt]: Done

**Pull request merged**